Java/small_game/Game.java

```diff
@@ -0,0 +1,122 @@
+/*
+        Simple program based on basics of java
+        i.e. Input/Output and simple decision making
+        with very well formatted Output
+*/
+import java.util.Scanner;
+public class Game
+{
+    public static void main(String args[])
+    {
+        Scanner in=new Scanner(System.in);
+        System.out.println("==========================================================");
+        System.out.println("welcome to WHO WHANT TO BE TOKEN KING");

```

Comment:
Typo Alert: "WHANT"
Also, either make this whole statement uppercase or capitalize on each word first character

Reply:
Okay will follow this